amgut/db/0042.sql

```diff
@@ -0,0 +1,129 @@
+CREATE TABLE ag."source" (
+	source_id            uuid DEFAULT ag.uuid_generate_v4() ,
+	source_name          varchar(100)  ,
+	ag_login_id          uuid  ,
```

Comment:
isn't that achieved by the later 
`ALTER TABLE ag.source_barcodes_surveys ADD CONSTRAINT fk_source_barcode FOREIGN KEY ( barcode ) REFERENCES barcodes.barcode( barcode );` like line?